src/index.json

```diff
@@ -975,6 +975,53 @@
                     "version": "0.1.0"
                 }
             }
+        ],
+        "hanaonazure": [
+            {
+                "filename": "hanaonazure-0.1.5-py2.py3-none-any.whl",
+                "sha256Digest": "0fb0aec0a18997877639d2674ff2cd908def9d0ff0e356f18c2067d1c7a07d76",
+                "downloadUrl": "https://github.com/Azure/azure-hanaonazure-cli-extension/releases/download/0.1.5/hanaonazure-0.1.5-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.minCliCoreVersion": "2.0.25",
```

Comment:
Have you tested out the extension for this version? If not, you may want to specify a later min-version like "2.0.43" as the CLI has changed a lot since.

Reply:
changed